src/main/java/com/github/jasminb/jsonapi/ResourceConverter.java

```diff
@@ -773,6 +774,13 @@ private ObjectNode getDataNode(Object object, Map<String, ObjectNode> includedCo
 		}
 		dataNode.set(ATTRIBUTES, attributesNode);
 
+		final PropertyNamingStrategy namingStrategy;
+		if (objectMapper.getPropertyNamingStrategy() != null) {
+			namingStrategy = objectMapper.getPropertyNamingStrategy();
+		}
+		else {
```

Comment:
Formatting:

```java
if () {
} else {
}
```

Also, please make sure to use tabs for indentation if not already.